### ISSUE TITLE
improve prediction accuracy

### ISSUE THREAD

**Issue by IntChem** [rxn4chemistry/paragraph2actions#3]
Hello, thansk for the nice job done! 
I just followed the training and finetuning(I used the same preproessed data used in training) commands. But the prediction accuracy is low, for example:

src_test.txt:
The reaction mixture is allowed to warm to room temperature and stirred overnight.
The combined organic layers were washed with brine, dried over sodium sulfate, filtered, and concentrated to yield 2.98 g of N-{2-[4-amino-7-[(6-aminohexyl)oxy]-2-(ethoxymethyl)-1H-imidazo[4,5-c]quinolin-1-yl]-1,1-dimethylethyl}-N-isopropylurea as a dark orange solid.
3-Amino-2,4-dichloro-benzamide (2.00 g, 9.8 mmol) in THF (45 mL) was added dropwise to LiAlH4 (1 M in THF, 24.4 mL) in THF (45 mL).
A further drop of methanesulfonyl chloride was added and the mixture continued to stir at RT for 30 min.

pred.txt:
STIR for overnight at room temperature; QUENCH with saturated aqueous NH4Cl solution.
PURIFY; COLLECTLAYER organic; PH with K2CO3 (0.5 g) to pH basic; EXTRACT with DCM (50 ml) 2 x; DRYSOLUTION over unknown; CONCENTRATE; YIELD yellow solid (101 mg, 31%).
ADD 3-Bromo-2-fluoroaniline (10 g, 52.63 mmol); ADD DCM (100 mL) under nitrogen.
STIR for hr at room temperature; ADD methanolic HCl.

Could you give some tips to improve prediction? Is it maily beacuse training data is insufficient, or the traning paramenters are not well set?

Thanks!

**Comment by avaucher** [comment]
Hi!
Did you train on the data files in this repository? If so, the reason is insufficient data. The files in this repository are provided as examples; the models presented in the paper were trained on more than 2M sentences.

**Comment by avaucher** [comment]
Closing after one year inactivity. Feel free to reopen if needed.
